frontend/public/components/factory/list-page.jsx

```diff
@@ -228,7 +228,7 @@ export const FireMan_ = connect(null, { filterList })(
         if (createProps.to) {
           createLink = (
             <Link className="co-m-primary-action" {...createProps}>
-              <Button variant="primary" id="yaml-create">
+              <Button variant="primary" id="yaml-create" data-test-id="yaml-create">
```

Comment:
I've always had mixed feelings about using `data-test` attribs, but if its something we already have going with protractor tests, its not something we need to rehash now.

Reply:
Yes, they are in use in our protractor tests.
Also `data-...` is Cypresses [best practice](https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements) for selection of elements.